docs/docs/getting-started/install.md

```diff
@@ -132,6 +132,138 @@ You need the following permissions on this project:
+                "ec2:DescribeSecurityGroups",
+                "ec2:CreateSubnet",
+                "ec2:CreateSecurityGroup",
+                "elasticloadbalancing:CreateTargetGroup",
+                "ec2:CreateInternetGateway",
+                "ec2:DescribeSubnets",
+                "elasticloadbalancing:DescribeTargetGroups",
+                "ec2:AttachInternetGateway",
+                "elasticloadbalancing:ModifyTargetGroupAttributes",
+                "ec2:DescribeInternetGateways",
+                "autoscaling:CreateAutoScalingGroup",
+                "iam:PassRole",
+                "ec2:CreateNatGateway",
+                "ec2:RevokeSecurityGroupEgress",
+                "elasticloadbalancing:DescribeTargetGroupAttributes",
+                "elasticloadbalancing:CreateLoadBalancer",
+                "ec2:DescribeNatGateways",
+                "elasticloadbalancing:DescribeTags",
+                "autoscaling:DescribeScalingActivities",
+                "ec2:CreateRouteTable",
+                "autoscaling:DescribeAutoScalingGroups",
+                "ec2:AuthorizeSecurityGroupIngress",
+                "ec2:AuthorizeSecurityGroupEgress",
+                "ec2:CreateRoute",
+                "ec2:AssociateRouteTable",
+                "elasticloadbalancing:DescribeTargetHealth",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "elasticloadbalancing:ModifyLoadBalancerAttributes",
+                "elasticloadbalancing:AddTags",
+                "elasticloadbalancing:DescribeLoadBalancerAttributes",
+                "elasticloadbalancing:CreateListener",
+                "elasticloadbalancing:DescribeListeners",
+                "logs:DeleteLogGroup",
+                "elasticloadbalancing:DeleteListener",
+                "ec2:DisassociateRouteTable",
+                "autoscaling:UpdateAutoScalingGroup",
+                "elasticloadbalancing:DeleteLoadBalancer",
+                "autoscaling:SetInstanceProtection",
+                "ec2:DescribeNetworkInterfaces",
+                "ec2:DeleteRouteTable",
+                "ec2:DeleteNatGateway",
+                "ec2:DetachInternetGateway",
+                "ec2:DisassociateAddress",
+                "ec2:ReleaseAddress",
+                "ec2:DeleteInternetGateway",
+                "ec2:DeleteSubnet",
+                "autoscaling:DeleteAutoScalingGroup",
+                "ec2:DeleteLaunchTemplate",
+                "elasticloadbalancing:DeleteTargetGroup",
+                "ec2:DeleteSecurityGroup",
+                "ec2:DeleteVpc"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+Follow Amazons' guide on [understanding](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies.html) and [managing policies](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_managed-vs-inline.html), and [roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_iam-tags.html).
```

Comment:
In my opinion the whole sentence is a bit awkward to read.
"Follow Amazon's guide on understanding and managing policies" by itself sounds fine, though I think both articles are actually about understanding the purpose of certain kinds of policies.

"Follow Amazon's guide on understanding and managing policies_, and roles_" sounds off to me. The comma implies a break in speech, yet "and roles" is meant to be read as "Follow Amazon's guide on understanding [...] roles" or "Follow Amazon's guide on managing [...] roles".

Not sure which one, though. And having two "ands" in the sentence is also not good, which makes it a confusing enumeration.

I would rewrite this, though I don't have a concrete suggestion as I don't fully know the articles and context of the linked articles to make a suggestion.

Reply:
I agree! I have removed the last link since it was off-context. The first two resources should provide enough information to know how policies work.